packages/messaging/src/controllers/base-controller.ts

```diff
@@ -43,11 +44,14 @@ export const TOKEN_EXPIRATION_MILLIS = 7 * 24 * 60 * 60 * 1000; // 7 days
 
 export abstract class BaseController implements FirebaseMessaging {
   INTERNAL: FirebaseServiceInternals;
+  readonly app!: FirebaseApp;
```

Comment:
Looks like you don't need the `!` since you are defining it in the constructor.

Reply:
Done.